### ISSUE TITLE
tensorflow logs.  I tensorflow/core/kernels/logging_ops.cc:

### ISSUE THREAD

**Issue by FMsunyh** [experiencor/keras-yolo3#54]
I get this issue, when i start to train. Can you have a look.
2018-05-09 17:29:43.413957: I tensorflow/core/kernels/logging_ops.cc:79] avg_iou                [11][0.64272052]                                                                    [35/1834]
2018-05-09 17:29:43.413986: I tensorflow/core/kernels/logging_ops.cc:79] avg_cat                [11][0]                                                                                      
2018-05-09 17:29:43.414010: I tensorflow/core/kernels/logging_ops.cc:79] recall50       [11][0]                                                                                              
2018-05-09 17:29:43.414033: I tensorflow/core/kernels/logging_ops.cc:79] recall75       [11][0]                                                                                              
2018-05-09 17:29:43.414057: I tensorflow/core/kernels/logging_ops.cc:79] count  [11][2]                                                                                                      
2018-05-09 17:29:43.414110: I tensorflow/core/kernels/logging_ops.cc:79] loss xy, wh, conf, class:      [11][7.87623692][69.0293198][37.2563438][10.1647]                                    
2018-05-09 17:29:43.421183: I tensorflow/core/kernels/logging_ops.cc:79] avg_obj                [22][0.318214476]                                                                            
2018-05-09 17:29:43.421234: I tensorflow/core/kernels/logging_ops.cc:79] avg_noobj              [22][0.240879714]                                                                            
2018-05-09 17:29:43.421277: I tensorflow/core/kernels/logging_ops.cc:79] avg_iou                [22][0.447552651]                                                                            
2018-05-09 17:29:43.421307: I tensorflow/core/kernels/logging_ops.cc:79] avg_cat                [22][0]                                                                                      
2018-05-09 17:29:43.421334: I tensorflow/core/kernels/logging_ops.cc:79] recall50       [22][0]                                                                                              
2018-05-09 17:29:43.421362: I tensorflow/core/kernels/logging_ops.cc:79] recall75       [22][0]                                                                                              
2018-05-09 17:29:43.421392: I tensorflow/core/kernels/logging_ops.cc:79] count  [22][1]                                                                                                      
2018-05-09 17:29:43.421456: I tensorflow/core/kernels/logging_ops.cc:79] loss xy, wh, conf, class:      [22][33.9489326][97.0432205][107.798874][5.14699745]                                 
2018-05-09 17:29:43.424161: I tensorflow/core/kernels/logging_ops.cc:79] avg_obj                [44][0.254808]                                                                               
2018-05-09 17:29:43.424205: I tensorflow/core/kernels/logging_ops.cc:79] avg_noobj              [44][0.193691671]                                                                            
2018-05-09 17:29:43.424235: I tensorflow/core/kernels/logging_ops.cc:79] avg_iou                [44][0.596340418]                                                                            
2018-05-09 17:29:43.424260: I tensorflow/core/kernels/logging_ops.cc:79] avg_cat                [44][0]

**Comment by FMsunyh** [comment]
I found the source code.
